drivers/retained_mem/retained_mem_rv3028/retained_mem_rv3028.c

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright (c) 2024 ANITRA system s.r.o.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define DT_DRV_COMPAT microcrystal_rv3028_retmem
+
+#include <zephyr/drivers/retained_mem.h>
+#include "rtc_rv3028.h"
+
+#include <zephyr/logging/log.h>
+LOG_MODULE_DECLARE(rv3028, CONFIG_RETAINED_MEM_LOG_LEVEL);
+
+#define RV3028_USER_RAM_SIZE (2 * sizeof(uint8_t))
```

Comment:
then this would be dropped, could have a assert to ensure it offset is not less than the first RAM address and offset + length are not above the second RAM address